### PR TITLE
Add ability to reassign perfherder alerts

### PR DESCRIPTION
**Pull request by wlach**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1190)

### CODE REVIEW

**Comment by wlach** [comment]
Various screenshots of feature in action:

screen shot 2015-12-03 at 4 56 47 pm
screen shot 2015-12-03 at 4 56 40 pm
screen shot 2015-12-03 at 4 56 29 pm

File: ui/js/controllers/perf/alerts.js [hunk 1]
```diff
@@ -19,16 +19,55 @@ perf.factory('PhAlerts', ['$http', 'thServiceDomain', function($http, thServiceD
...
     };
 }]);
 
+perf.controller(
+    'ReassignAlertsCtrl',
+    function($scope, $modalInstance, $http, $q, alertSummary, PhAlerts) {
+        var alerts = _.where(alertSummary.alerts, {'selected': true});
+        $scope.reassignAlerts = function() {
+            var revisedSummaryId = parseInt(
+                $scope.reassignAlertId.newAlertSummaryId.$modelValue);
```
Comment (jmaher) [inline ui/js/controllers/perf/alerts.js]: what if revisedSummaryID is invalid or not an int?

Comment (wlach) [inline ui/js/controllers/perf/alerts.js]: We don't allow submitting the form if the input isn't a natural number.

Comment (wlach) [inline ui/js/controllers/perf/alerts.js]: Though to be fair, the UI doesn't have any error handling right now. We need to fix that, but I'm not sure if it should block landing this.

File: ui/partials/perf/alertsctrl.html [hunk 2]
```diff
@@ -85,6 +99,14 @@
           </td>
         </tr>
       </table>
+      <div style="padding-left: 8px;">
+        <button class="btn btn-default" role="button" ng-show="anySelected(alertSummary.alerts)"
+                ng-click="reassignAlerts(alertSummary)">Reassign</button>
+        <button class="btn btn-default" role="button"
+                ng-show="anySelected(alertSummary.alerts)"
+                ng-disabled="!anyReassignedAndSelected(alertSummary.alerts)"
+                ng-click="unassignAlerts(alertSummary)">Unassign</button>
```
Comment (jmaher) [inline ui/partials/perf/alertsctrl.html]: so there is an unassign button?

**Pull request merged**